ivadomed/utils.py

```diff
@@ -581,6 +639,10 @@ def dice_score(im1, im2):
 
 
 def hausdorff_score(prediction, groundtruth):
+    if len(prediction.shape) == 4:
+        l, h, d, w = prediction.shape
+        prediction = prediction.reshape((h, l * d, w))
+        groundtruth = groundtruth.reshape((h, l * d, w))
```

Comment:
Not sure here, could you please explain?

Reply:
I added a comment to explain! Tell me if you need more explanation